app/src/main/java/com/cornellappdev/transit/ui/screens/DetailsScreen.kt

```diff
@@ -41,20 +38,17 @@ import com.google.maps.android.compose.rememberCameraPositionState
  */
 @OptIn(ExperimentalMaterial3Api::class, ExperimentalPermissionsApi::class)
 @Composable
-fun DetailsScreen(navController: NavHostController, homeViewModel: HomeViewModel) {
+fun DetailsScreen(navController: NavHostController, routeViewModel: RouteViewModel) {
 
     val permissionState = rememberPermissionState(Manifest.permission.ACCESS_FINE_LOCATION)
 
     //Map camera
     val cameraPositionState = rememberCameraPositionState {
-        position = CameraPosition.fromLatLngZoom(homeViewModel.defaultIthaca, 12f)
+        position = CameraPosition.fromLatLngZoom(routeViewModel.defaultIthaca, 12f)
     }
```

Comment:
Hmm so I assume there'll have to be something done here to make the zoom actually zoom on the route? Since it's super zoomed out right now.

Reply:
Oh I didn't notice this. Will make an issue for it!